pkg/services/ngalert/api/api_prometheus.go

```diff
@@ -69,7 +86,7 @@ func (srv PrometheusSrv) RouteGetAlertStatuses(c *contextmodel.ReqContext) respo
 		})
 	}
 
-	return response.JSON(http.StatusOK, alertResponse)
+	return data
```

Comment:
All code above this line I'm not familiar with I'm afraid, so just wanted to check that you've tested this and made sure it works as it did before?